DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/ClassUsageAnalysis.scala

```diff
@@ -35,28 +31,41 @@ import org.opalj.value.ValueInformation
  * [[ClassUsageAnalysis.analysisSpecificParametersDescription]].
  *
  * @author Patrick Mell
- * @author Dominik Helm
  */
 object ClassUsageAnalysis extends ProjectAnalysisApplication {
 
-    private type V = DUVar[ValueInformation]
-
     implicit val logContext: GlobalLogContext.type = GlobalLogContext
 
     override def title: String = "Class Usage Analysis"
 
     override def description: String = {
-        "Analyzes a project for how a particular class is used within it, i.e., which methods " +
-            "of instances of that class are called"
+        "Analysis a project for how a particular class is used, i.e., which methods are called " +
+            "on it"
     }
 
+    /**
+     * The fully-qualified name of the class that is to be analyzed in a Java format, i.e., dots as
+     * package / class separators.
+     */
+    private val className = "java.lang.StringBuilder"
```

Comment:
These two values should obviousbly be command line parameters, why have they been turned into hardcoded values?

Reply:
This class was modified on accident and is not inherently connected to the string analysis (in such a way that it requires this class to be modified).